### PR TITLE
=sbt Apply the build sbt messages to all akka modules.

### PR DESCRIPTION
**Pull request by He-Pin**
~~Sorry, adding only to `sbt` only will not work when reloading in the subprojects.~~

Only available on root project and disable it on subprojects.

### CODE REVIEW

**Review by raboof — APPROVED** [review]
I think having it just for the root project might actually be sufficient? But I'm not opposed to having it for subprojects as well.

**Comment by hepin1989** [comment]
If needs to be silenced then the on_load messages should be set to empty then.

File: build.sbt [hunk 1]
```diff
@@ -624,6 +597,7 @@ lazy val serialversionRemoverPluginSettings = Seq(
 def akkaModule(name: String): Project =
   Project(id = name, base = file(name))
     .enablePlugins(ReproducibleBuildsPlugin)
+    .disablePlugins(WelcomePlugin)
```
Comment (hepin1989) [inline build.sbt:600]: disabled on sub projects

**Review by raboof — APPROVED** [review]
ah, now I understand the problem with the current main branch: it currently shows the default 'example' welcome message for subprojects.

Disabling for subprojects seems like a great approach!

**Comment by hepin1989** [comment]
Yes，I missed that part....

**Pull request merged**